### PR TITLE
Add search dumb component for mobile

### PR DESCRIPTION
**Pull request by MarcoPolo**
@keybase/react-hackers 

Search screen for mobile

The preview is showing some states that don't make sense together, but just there to show how different rows would look without having a bunch of different mocks.

Also the twitter icon is not colored. Not sure if we want to make a separate image icon for these services that are colored correctly or try to color the font icons ourselves. I'm leaning towards using images because that'll keep our colors separate from service colors and it'll work for multicolor logos (e.g. reddit).

image

image

## Todo:
Service images or colored icons

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
:mag_right: No visual changes found as a result of these commits.

**Comment by chromakode** [comment]
+1 to including multicolor images for the services.

File: shared/search/user-search/render.native.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+import React, {Component} from 'react'
+import {Avatar, Box, Icon, Input, Text, ListItem} from '../../common-adapters'
+import {globalStyles, globalColors} from '../../styles/style-guide'
+
+import type {Props, SearchResult} from './render'
+import type {Combo} from '../../constants/types/more'
+import type {Props as TextProps} from '../../common-adapters/text'
+
+function EmboldenTextMatch ({text, match, style, textType, emboldenStyle}: {text: string, match: string, emboldenStyle: Object, style: Object, textType: TextProps.type}) {
+  const indexOfMatch = text.indexOf(match)
```
Comment (chromakode) [inline shared/search/user-search/render.native.js]: In the mockup, "Malg" in "John Malg" is bolded. Should this match be case-insensitive?

**Comment by MarcoPolo** [comment]
- [ ] Make John Malg be centered

**Comment by chrisnojima** [comment]
the icons already exist in the style sheet in zeplin

File: shared/search/user-search/render.native.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+      break
+    case 'external':
+      icon = <Icon type={result.icon} style={iconStyle} />
+      body = <ExternalResultBody username={result.username} searchText={searchText} />
+      break
+  }
+
+  // Align the body to the middle of the original 32 row so the extra info doesn't push the username down
+  const alignedBody = (
+    <Box style={{flex: 2}}>
```
Comment (chromakode) [inline shared/search/user-search/render.native.js]: Is `flex: 2` necessary here? Will `1` suffice?

Comment (chrisnojima) [inline shared/search/user-search/render.native.js]: why even 1? doesn't body take up all the space anyways?

**Comment by keybase-ci-visdiff** [comment]
:mag_right: No visual changes found as a result of these commits.

File: shared/search/user-search/render.native.js (hunk 2 above)
Comment (MarcoPolo) [inline shared/search/user-search/render.native.js]: It's put along side the extra info to the right (e.g. twitter name or keybase full name). So you want this to use up all the space with the extra info to the right

Comment (MarcoPolo) [inline shared/search/user-search/render.native.js]: This picture may help:

screen shot 2016-05-26 at 6 52 06 pm


the `alignedBody` is yellow here.

The list-item body is red

**Comment by keybase-ci-visdiff** [comment]
:mag_right: No visual changes found as a result of these commits.

File: shared/search/user-search/render.native.js (hunk 2 above)
Comment (MarcoPolo) [inline shared/search/user-search/render.native.js]: talked with mgood, and he's right 1 will suffice!

**Comment by keybase-ci-visdiff** [comment]
:mag_right: No visual changes found as a result of these commits.

**Comment by chrisnojima** [comment]
👍

**Pull request merged**